EIPS/eip-4750.md

```diff
@@ -97,6 +98,16 @@ Under `PC_post_instruction` we mean the PC position after the entire immediate a
 
 6. Sets `current_section_index` to `code_section_index` and `PC` to `0`, and execution continues in the called section.
 
+#### `TAILCALLF`
+
+1. Has one immediate argument,`code_section_index`, encoded as a 16-bit unsigned big-endian value.
+2. If data stack has less than `caller_stack_height +  type[code_section_index].inputs` items, execution results in exceptional halt.
+3. If data stack has more than `caller_stack_height +  type[code_section_index].inputs` items, discards the items between `caller_stack_height` and top `type[code_section_index].inputs`, so that there are exactly `caller_stack_height +  type[code_section_index].inputs` items left.
+4. Charges 6 gas.
```

Comment:
Tentative price aiming for between `CALLF` and `CALLF + RETF`

Reply:
Actually it seems it should be cheaper because it has 2 less steps compared to `CALLF` (the stack checks).

Reply:
But it has step 3, which is automatic stack clearing that CALLF doesn't need.

Reply:
So kind of like equals out then? I'm fine with either, this one needs to be benchmarked.